docs/samples/Microsoft.ML.Samples/Dynamic/LogisticRegression.cs

```diff
@@ -63,7 +63,7 @@ public static void Example()
                 .Append(ml.Transforms.Text.FeaturizeText("TextFeatures", "Text"))
                 .Append(ml.Transforms.Concatenate("Features", "TextFeatures", "age", "fnlwgt",
                     "education-num", "capital-gain", "capital-loss", "hours-per-week"))
-                .Append(ml.BinaryClassification.Trainers.LogisticRegression());
+                .Append(ml.BinaryClassification.Trainers.LbfgsLogisticRegression());
```

Comment:
How do you feel about making the other calibrated linear trainers, like SDCA into `XyzLogisticRegression()`.

Reply:
We can always add that at a later time.